internal/js/modules/k6/browser/common/frame.go

```diff
@@ -1084,6 +1080,29 @@ func (f *Frame) GetByRole(role string, opts *GetByRoleOptions) *Locator {
 	return f.Locator(builder.String(), nil)
 }
 
+// isQuotedText returns true if the string is a quoted string.
+// This is used to determine if the string will be used to match
+// a string instead of as a regex in the getBy* APIs.
+func isQuotedText(s string) bool {
+	return len(s) > 1 && s[0] == '\'' && s[len(s)-1] == '\''
+}
```

Comment:
This might fail if Unicode characters are passed. See: https://go.dev/blog/strings.